reporter/build.gradle

```diff
@@ -4,6 +4,20 @@ apply plugin: 'application'
 applicationName = 'reporter'
 mainClassName = 'com.here.ort.reporter.Main'
 
+sourceSets {
+    main {
+        output.dir("$buildDir/generated-resources/main", builtBy: 'copyWebAppTemplate')
+    }
+}
+
+task copyWebAppTemplate(type: Copy, dependsOn: ':reporter-web-app:build') {
```

Comment:
Nit: We usually pass the task *object*, not its string representation, to `dependsOn`. Not sure how the syntax is for tasks of another project, though.

Reply:
Should be `project(':reporter-web-app').tasks.build`. Would you prefer that? I find `:reporter-web-app:build` easier to read.

Reply:
No. If it's that convoluted I prefer the current syntax.